frame/oracle/src/benchmarking.rs

```diff
@@ -41,10 +38,10 @@ benchmarks! {
 	add_asset_and_info {
 		let caller = T::AddOracle::successful_origin();
 		let asset_id = 1;
-		Validated::new(Percent::from_percent(80)).unwrap(),
-		Validated::new(3).unwrap(),
-		Validated::new(5).unwrap(),
-		Validated::<T::BlockNumber, ValidBlockInterval<T::StalePrice>>::new(5).unwrap(),
+		Validated::new(Percent::from_percent(80)).unwrap();
```

Comment:
in lending benches are part of tests and run as part, so these will never break

Reply:
What do you mean by this?